protocol/bitcoin_scanner.go

```diff
@@ -9,10 +9,10 @@ import (
 type BitcoinProtocolScanner struct {
 }
 
-func (rps *BitcoinProtocolScanner) ScanProtocol(hiddenService string, proxyAddress string, report *report.OnionScanReport) {
+func (rps *BitcoinProtocolScanner) ScanProtocol(hiddenService string, os *ProtocolConfig, report *report.OnionScanReport) {
```

Comment:
Go styling: ProtocolConfig variable shouldn't shorten to os. (applies to other lines too)